app/services/zendesk_service.rb

```diff
@@ -0,0 +1,45 @@
+class ZendeskService
+  attr_accessor :ticket
+
+  class << self
+    def send!(ticket)
+      ZendeskService.new(ticket).send!
```

Comment:
```suggestion
      new(ticket).send!
```
It's already in the ZendeckService class scope, so you don't need to re-reference it